go-fuzz-build/main.go

```diff
@@ -737,18 +751,41 @@ func (c *Context) instrumentPackages(blocks *[]CoverBlock, sonar *[]CoverBlock)
 }
 
 func (c *Context) copyDir(dir, newDir string) {
-	c.mkdirAll(newDir)
 	files, err := ioutil.ReadDir(dir)
 	if err != nil {
 		c.failf("failed to scan dir '%v': %v", dir, err)
 	}
+	c.mkdirAll(newDir)
 	for _, f := range files {
+		src := filepath.Join(dir, f.Name())
+		dst := filepath.Join(newDir, f.Name())
 		if f.IsDir() {
-			c.copyDir(filepath.Join(dir, f.Name()), filepath.Join(newDir, f.Name()))
+			c.copyDir(src, dst)
 			continue
 		}
+		c.copyFile(src, dst)
+	}
+}
+
+func (c *Context) copyNotPkgDir(dir, newDir string) {
+	files, err := ioutil.ReadDir(dir)
+	if err != nil {
+		c.failf("failed to scan dir '%v': %v", dir, err)
+	}
+	for _, f := range files {
+		if strings.HasSuffix(f.Name(), ".go") {
```

Comment:
I don't understand why this check is here. Can you expand the comment, please?

Reply:
Commented at https://github.com/dvyukov/go-fuzz/pull/311/commits/932e472456e74cf8b35a42c349a49c41e62bde11